Firmware/MotorControl/motor.cpp

```diff
@@ -150,8 +150,7 @@ float Motor::get_inverter_temp() {
     return horner_fma(normalized_voltage, thermistor_poly_coeffs, thermistor_num_coeffs);
 }
 
-bool Motor::update_thermal_limits() {
-    float fet_temp = get_inverter_temp();
+bool Motor::update_thermal_limits(float fet_temp) {
```

Comment:
why was `float fet_temp = get_inverter_temp();` moved out of here instead of doing `inverter_temp_ = fet_temp;` here? Not that it makes a big difference.

Reply:
I'm not a big fan of putting HWIO operations inside logical functions.  IMO it's better architecture to have a clear delineation:

- Read Inputs
- Update Logic
- Write Outputs